diff --git a/monique_gui.pro b/monique_gui.pro
index 9fd837e..e085e81 100644
--- a/monique_gui.pro
+++ b/monique_gui.pro
@@ -11,7 +11,7 @@ QT += gui widgets
 #DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x060000    # disables all the APIs deprecated before Qt 6.0.0
 
 # Get the current Git commit hash
-DEFINES += MONIQUE_GUI_GIT_COMMIT_HASH="\\\"$(shell git log -1 --format=%H)\\\""
+DEFINES += MONIQUE_GUI_GIT_COMMIT_HASH="\\\"$$system(cd ../git; git log -1 --format=%H)\\\""
 
 # Paths
 INCLUDEPATH += include
